packages/core/package.json

```diff
@@ -1,20 +1,17 @@
 {
-  "name": "typed-css-core",
+  "name": "@glennsl/bs-typed-css-core",
   "version": "0.1.0",
   "scripts": {
-    "build": "bsb -make-world",
-    "start": "bsb -make-world -w",
-    "clean": "bsb -clean-world"
+    "build": "../../node_modules/.bin/bsb -make-world",
```

Comment:
Why is this necessary? Wouldn't it be better to keep `bs-platform` as a devDependency here to avoid this? lerna/yarn workspaces should make sure it's symlinked to the root `node_modules` and therefore only installed once

Reply:
Sure we can also do that if you prefer. I need to enable yarn workspaces though